app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
                 <category android:name="android.intent.category.LAUNCHER"/>
+                <action android:name="android.intent.action.VIEW"/>
```

Comment:
Curious 🤔 Why is this needed?

Reply:
![screen shot 2018-10-23 at 9 33 21 am](https://user-images.githubusercontent.com/8434572/47366461-ba691280-d6ab-11e8-8648-93cb2e977e96.png)

This lint warning was highlighting the whole `AndroidManifest.xml`